doc/src/site/sphinx/troubleshooting.rst

```diff
@@ -0,0 +1,101 @@
+Production Checklist & Troubleshooting Guide
+********************************************
+
+Production Checklist
+====================
+
+In this section we provide you the production environment basic settings.
+
+First of all, you should check the different systems and services availability in your environment.
+
+Then to configure our service you need to take a look at two files.
+
+application.conf
+----------------
+Where? -> /etc/sds/sparkta/application.conf
+
+What should you check?
+
+#. Network interfaces and ports
+#. Zookeeper configuration
+#. Hdfs paths
+#. Resource manager configuration (mesos/yarn/standalone/local)
+#. Spark configurations
+
+run.sh
+------
+Where? -> /opt/sds/sparkta/bin/run.sh
+
+What should you check?
+
+#. Memory configuration
+#. Jmx port
+#. Other Java opts
+
+Finally you should check your inputs and outputs availability and their configuration in your policies.
+
+
+Troubleshooting
+===============
+
+This section tries to group the common errors our users find. Most of them are easily fixed checking the logs.
+Sparta logs are at `/var/log/sds/sparkta/sparkta.[out|err]` . Please if you find an error that belongs to Sparkta don't
+hesitate in open an issue at |github_dl|
+
+.. |github_dl| raw:: html
+
+   <a href="https://github.com/Stratio/Sparkta/issues" target="_blank">GitHub</a>
+
+
+
+**Error when a cube has a dimension of type DateTime**::
+
+    java.lang.ClassCastException: java.lang.Long cannot be cast to java.util.Date
+            at com.stratio.sparkta.plugin.field.datetime.DateTimeField.precisionValue(DateTimeField.scala:58)
+            at com.stratio.sparkta.aggregator.CubeOperations$$anonfun$extractDimensionsAggregations$1$$anonfun$1$$anonfun$apply$1.apply(CubeMaker.scala:75)
+            at com.stratio.sparkta.aggregator.CubeOperations$$anonfun$extractDimensionsAggregations$1$$anonfun$1$$anonfun$apply$1.apply(CubeMaker.scala:74)
+            at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
+            at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
+            at scala.collection.immutable.List.foreach(List.scala:318)
+
+When one of your dimensions is of the type DateTime the field this dimension is pointing to must be a date. The
+solution is to `parse <transformations.html#_datetime-transformation-label>`__ this field to be in Date format.
+
```

Comment:
If the type of the dimensions has to be a date, you should use the right parser (in this case you must use `parse <transformations.html#_datetime-transformation-label>`)